docs/source/whitepaper.rst

```diff
@@ -66,6 +65,7 @@ The parameters that need to be specified by the users are:
 - recipient address on target rollup
 - amount
 - validity period of request
+- expiry block number
```

Comment:
I think this is not necessary after today's discussion. The expiry block number will be checked implicitly by the contracts

Reply:
ok